docs/source/analyst/example_usage.ipynb

```diff
@@ -103,36 +185,41 @@
    ],
    "source": [
     "query_id = flowclient.run_query(conn, query_spec)\n",
-    "print(query_id)"
+    "query_id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check status of query"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 10,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "(False, <Response [202]>)"
+       "(True, <Response [303]>)"
```

Comment:
Isn't the output of `get_status()` supposed to be a string now? Looks like some of the cells need to be re-executed maybe?

Reply:
Yes, that output was left over from before I added `get_status()`. I've cleared all outputs now, and checked that the output of `get_status()` is correct when it runs during the docs build.